sfn-query-large-datasets-cdk/README.md

```diff
@@ -0,0 +1,110 @@
+# Query large datasets (Amazon Athena, Amazon S3, AWS Glue, Amazon SNS)
+
+**Resources used**  
+
+| Resource Type | Name    |
+|------------------|--------------------|
+| Athena Workgroup | AthenaWorkgroup    |
+| State Machine    | AthenaStateMachine |
+| SNS Topic        | SNSTopic           |
+| S3 Bucket        | DataBucket         |
+| Glue Database    | GlueDatabase       |
+| Glue Crawler     | GlueCrawler        |  
+  
+## Start an Athena query
+
+This workflow demonstrates how to ingest a large data set in Amazon S3 and partition it through AWS Glue Crawlers, then execute Amazon Athena queries against that partition. Deploying this sample project creates an AWS Step Functions state machine, an Amazon S3 Bucket, an AWS Glue crawler, and an Amazon SNS topic.
```

Comment:
I know this is coming from the terraform variant, but for me "and partition it through AWS Glue crawlers" is not 100% accurate. Data has been already partitioned by leveraging different S3 prefixes and the crawler is capable of interpreting it correctly to add the corresponding partitions to the table definition in the data catalog so future queries against run faster.

Reply:
It's not 100% accurate describing it, but it's still correct and also the same as the [AWS documentation](https://docs.aws.amazon.com/step-functions/latest/dg/sample-query-large-datasets.html).